components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m

```diff
@@ -92,7 +92,8 @@ - (UIView *)buttonBar:(MDCButtonBar *)buttonBar
   MDCButtonBarButton *button = [[MDCButtonBarButton alloc] init];
   [button setBackgroundColor:[UIColor clearColor] forState:UIControlStateNormal];
   button.disabledAlpha = kDisabledButtonAlpha;
-
+  button.inkColor = buttonBar.inkColor ?: buttonBar.inkColor;
```

Comment:
I think the latter part of this expression is meant to be `button.inkColor`.

Reply:
Done.